README

```diff
@@ -156,21 +156,47 @@ REQUIREMENTS:
         dracut (optional)
         PolicyKit (optional)
 
-        When building from git, the following tools are needed:
+        Two build systems are supported: meson + ninja-build and autools + make.
+
+        The following tools are needed with both systems:
 
         pkg-config
-        docbook-xsl
-        xsltproc
+        gperf >= 3.1
+        docbook-xsl (optional, required for documentation)
+        xsltproc    (optional, required for documentation)
+        python-lxml (optional, required to build the indices)
+
+        When building with meson, python and ninja-build are required.
```

Comment:
Ah, it's ninja-build that uses python2 for something (http://pkgs.fedoraproject.org/cgit/rpms/ninja-build.git/tree/ninja-build.spec#n12).